uefi-test-runner/src/proto/console/gop.rs

```diff
@@ -59,6 +62,12 @@ fn fill_color(gop: &mut GraphicsOutput) {
 
 // Draw directly to the frame buffer.
 fn draw_fb(gop: &mut GraphicsOutput) {
+    // The `virtio-gpu-pci` graphics device we use on aarch64 doesn't
+    // support `PixelFormat::BltOnly`.
+    if cfg!(target_arch = "aarch64") {
+        return;
```

Comment:
So the test now checks if the protocol is available on aarch64 but doesn't actually test it, right?